data/data-pipeline/data_pipeline/etl/score/schemas/datasets.py

```diff
@@ -37,30 +37,46 @@ class LoadField:
                 short_name (str): Used in conjunction with the dataset's `short_name` for files where short names are needed.
                 df_field_name (str): Name for the field in the etl class.
                 long_name (str): Column name for the dataset's output csv.
-                field_type (FieldType): An enum that dictates what type of field this is. This will be used on the `etl_score_post`
-                for the data manipulation. The `by_value` metadata prop will load the field type's Enum value instead of the index, i.e. "string"
-                and not STRING reverse_percentile (Optional bool): An optional bool to denote this field to be a reverse_percentile.
-                TODO: data/data-pipeline/data_pipeline/etl/score/etl_score.py:477
+                field_type (FieldType): An enum that dictates what type of field this is. This will be used on the `etl_score_post` for the
+                    data manipulation. The `by_value` metadata prop will load the field type's Enum value instead of the index, i.e. "string"
+                    and not STRING
+                number_of_decimals_in_output (int): Used to represent number of decimals in side effects, like Excel.
```

Comment:
:tada:

Reply:
Can you specify what the default value is if number_of_decimals_in_output is not specified? 

Also - many of your doc strings are for optional fields but don't include the `Optional` note (e.g. `(Optional bool)`). Could you add that?